src/osdc/Objecter.h

```diff
@@ -1610,8 +1610,6 @@ class Objecter : public md_config_obs_t, public Dispatcher {
 
     ceph::buffer::list filter;
 
-    ceph::buffer::list extra_info;
```

Comment:
this field was added when we added the pgls. see https://github.com/ceph/ceph/commit/a03f85a8e7fab296ea2df70a929a1c5e4aa0f7fb#diff-847b3447bebb2f031678eef0a2aea60dR2764

i am adding Sam and David as reviewers for more inputs.

Reply:
i think its only usecase is for holding the objects' `inode_backpointer_t` when handling "parent" pgls-filter, which is supposed to be used by cephfs client to enumerate all objects whose parent inode is identical to the one packed as the input buffer parameter.

but the question is who is still using the "parent" pgls filter, after reading branches from firefly to nautilus, i could not find its caller. not to mention the caller who is inspecting the returned `inode_backpointer_t`.

@batrick could you please help confirm this?

Reply:
Patrick mentioned that it could be used by "cephfs-data-scan". and we could schedule a run with "--suite fs --filter data-scan" to see if it's broken if we drop the this extra_info,  or simply ditch the "parent" filter.

Reply:
http://pulpito.ceph.com/kchai-2019-08-15_07:03:54-fs-wip-kefu-testing-2019-08-15-1125-distro-basic-smithi/